src/index.scss

```diff
@@ -0,0 +1,305 @@
+$grid-gutter-width: 30px;
```

Comment:
This is way too much code to be in a single file. Please split it, ideally with separate style files for each of the React components they correspond to.